pkg/maglev/maglev.go

```diff
@@ -28,16 +28,26 @@ const (
 
 	// seed=$(head -c12 /dev/urandom | base64 -w0)
 	DefaultHashSeed = "JLfvgnHc2kaSUFaI"
+
+	// DefaultMaxBackends is the default number of backends per service
+	// supported in Maglev. This value is used in initializing the Maglev
+	// permutations.
+	DefaultMaxBackends = 5000
 )
 
 var (
 	seedMurmur uint32
 
 	SeedJhash0 uint32
 	SeedJhash1 uint32
+
+	maxBackends uint64
+
+	// permutation is the slice containing the Maglev permutation calculations.
+	permutation []uint64
 )
 
-func InitMaglevSeeds(seed string) error {
+func InitMaglevSeeds(seed string, m, backends uint64) error {
```

Comment:
Nit: `m` is not a very descriptive name. Maybe add a short godoc comment here explaining what `m` is (and while at it also for the other parameters)? Or alternatively rename it so something more descriptive?